internal/runners/show/show.go

```diff
@@ -46,86 +63,102 @@ func New(prime primeable) *Show {
 
 // Run is the primary show logic.
 func (s *Show) Run(params Params) error {
-	logging.Debug("Execute")
-
-	if s.project == nil {
-		return locale.NewError("err_no_projectfile")
-	}
+	logging.Debug("Execute show")
 
-	pj := s.project
+	var (
+		owner       string
+		projectName string
+		projectURL  string
+		events      []string
+		scripts     map[string]string
+		err         error
+	)
 
 	if params.Remote != "" {
-		path := params.Remote
-		projectFilePath := filepath.Join(params.Remote, constants.ConfigFileName)
-
-		if _, err := os.Stat(path); err != nil {
-			return locale.WrapError(
-				err,
-				"err_state_show_path_does_not_exist",
-				"Directory does not exist.",
-			)
+		namespaced, fail := project.ParseNamespace(params.Remote)
+		if fail != nil {
+			return locale.WrapError(fail.ToError(), "err_show_parse_namespace", "Invalid remote argument, must be of the form <Owner>/<Project>")
 		}
 
-		if _, err := os.Stat(projectFilePath); err != nil {
-			return locale.WrapError(
-				err,
-				"err_state_show_no_config",
-				"activestate.yaml file not found in the given location.",
-			)
+		owner = namespaced.Owner
+		projectName = namespaced.Project
+	} else {
+		if s.project == nil {
+			return locale.NewError("err_no_projectfile")
 		}
 
-		projectFile, err := projectfile.Parse(projectFilePath)
+		owner = s.project.Source().Owner
+		projectName = s.project.Source().Name
+		projectURL = s.project.Source().Project
```

Comment:
I don't believe you need the source projectfile for any of these, just use the methods on `s.project`, eg. `s.project.Name()`.